src/js/locale/en.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I'm just wondering what conventions we want to set for this going forward, do we want to namespace these under `"components"`  or are we happy with them like this.
@ThomasPalamara

Reply:
I don't think it's important to have a namespace. There is very little text as you can see and they are all in components